examples/bidders/a9/script.js

```diff
@@ -1,15 +1,41 @@
-import { AdEngine, context, events } from '@wikia/ad-engine';
-import { utils } from '@wikia/ad-products';
+import { AdEngine, context, events, utils } from '@wikia/ad-engine';
+import { utils as adProductsUtils } from '@wikia/ad-products';
 import { bidders } from '@wikia/bidders';
 
 import customContext from '../../context';
 import '../../styles.scss';
 
+const optIn = utils.queryString.get('tracking-opt-in-status') !== '0';
```

Comment:
Do we mention in the example page content we can use the param?

Reply:
Done :+1: